server/handlers/find.js

```diff
@@ -120,19 +120,13 @@ export const findNotificationByRecipient = async recipient => {
  * @return {Promise<UserComment> | NotificationNotFound}
  */
 export const findCommentByPostId = async _id => {
-  let comment = [];
-  let error = {};
-  comment = await Comment.find({
+  let commentList = [];
+  commentList = await Comment.find({
```

Comment:
you could just directly return Comment.find({
blablabla
}).blabla;

Reply:
because this just waits for a promise to return

Reply:
fixed